guice/src/main/java/com/flipkart/gjex/guice/module/DashboardModule.java

```diff
@@ -139,13 +140,21 @@ Server getDashboardJettyServer(@Named("Dashboard.service.port") int port,
 	@Singleton
 	Server getAPIJettyServer(@Named("APIVanillaJettyServer")Server server,
 							@Named("ApiServletContext") ServletContextHandler context,
-							@Named("HealthCheckResourceConfig")ResourceConfig healthcheckResourceConfig,
+							@Named("HealthCheckResourceConfig") ResourceConfig healthCheckResourceConfig,
+							@Named("RotationManagementResourceConfig") ResourceConfig rotationManagementResourceConfig,
 							@Named("TracingResourceConfig")ResourceConfig tracingResourceConfig,
 							@Named("TracingSamplerHolder")TracingSamplerHolder tracingSamplerHolder,
 							@Named("JSONMarshallingProvider")JacksonJaxbJsonProvider provider) throws URISyntaxException, UnknownHostException {
-		healthcheckResourceConfig.register(provider);
-		ServletHolder healthcheckServlet = new ServletHolder(new ServletContainer(healthcheckResourceConfig));
-		context.addServlet(healthcheckServlet, "/healthcheck"); // registering Health Check servlet under the /healthcheck path
+		healthCheckResourceConfig.register(provider);
+		ServletHolder healthCheckServlet =
+				new ServletHolder(new ServletContainer(healthCheckResourceConfig));
+		context.addServlet(healthCheckServlet, "/healthcheck"); // registering Health Check servlet under the /healthcheck path
+
+		rotationManagementResourceConfig.register(provider);
+		ServletHolder rotationManagementServlet =
+				new ServletHolder(new ServletContainer(rotationManagementResourceConfig));
+		context.addServlet(rotationManagementServlet, "/rotation"); // registering Rotation
```

Comment:
You will have to use wildcard here if you want to register multiple API paths to this servlet .

Reply:
updated path as /rotation/*